soc/arm/renesas_rcar/gen3/pfc-r8a77951.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _PFC_RCAR_R8A77951_
+#define _PFC_RCAR_R8A77951_
+#include "pfc-rcar.h"
+
+static const struct rcar_pin can0_data_a[] = {
```

Comment:
We should encode these in devicetree.  Look at what STM32, NXP Kinetis and Atmel are doing here.

Reply:
ok will do, thanks for the review @galak  !